org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/XMLCompletions.java

```diff
@@ -288,7 +294,83 @@ public CompletionList doComplete(DOMDocument xmlDocument, Position position, Sha
 			return completionResponse;
 		} finally {
 			collectSnippetSuggestions(completionRequest, completionResponse);
+			// Manage itemDefaults
+			CompletionItemDefaults itemDefaults = new CompletionItemDefaults();

```

Comment:
As code from XMLCompletion is very big, could you please create an utility class like CompletionItemDefaultsUtils with only public static method `CompletionItemDefaultsUtils.process(completionRequest, completionResponse);`